src/main/java/com/icthh/xm/ms/entity/repository/XmEntityProjectionRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.icthh.xm.ms.entity.repository;
+
+import com.icthh.xm.ms.entity.projection.XmEntityIdKeyTypeKey;
+import com.icthh.xm.ms.entity.projection.XmEntityStateProjection;
+
+public interface XmEntityProjectionRepository {
+
+    XmEntityStateProjection findStateProjectionByKey(String key);
```

Comment:
why this method was moved to separate repository? correct palce for this method it's XmEntityRepository